core/server/data/importer/import-manager.js

```diff
@@ -390,7 +390,7 @@ class ImportManager {
             return await this.generateReport(importResult);
         } finally {
             // Step 5: Cleanup any files
-            this.cleanUp();
+            await this.cleanUp();
```

Comment:
The `await` here is changing the behavior of the importer from "fire-and-forget" cleanup (does not block the request processing) to wait for a full cleanup and then continue the request processing. It would be beneficial to verify if we gain anything from this change, otherwise don't see a good reason to change the current flow.

Was there a test for this change with a bigger import? How does the execution time compare. 

I see the good intention here, but wanted to make sure we don't accidentally introduce a bottleneck without a verification :upside_down_face:

Reply:
I understand! I'll check and remove it probably :)